json-schemas/workflowRunsAggregateRequest.json

```diff
@@ -0,0 +1,130 @@
+{
+    "type": "object",
+    "properties": {
+        "workflowRunIds": {
+            "type": "object",
+            "properties": {
+                "short-read-mngs": {
+                    "type": "array",
+                    "items": {
+                        "type": "integer"
```

Comment:
might be good to just start sending `string`s now, since NextGen will expect them